build/GenerateResxSource.targets

```diff
@@ -1,8 +1,13 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <PropertyGroup>
+    <CustomizeXlfSourceNamesDependsOn Condition="'$(SkipCustomizeXlfSourceNames)'!='true'">_CustomizeResourceNames</CustomizeXlfSourceNamesDependsOn>
+  </PropertyGroup>
+
   <Target Name="_CustomizeXlfSourceNames"
-          DependsOnTargets="_CustomizeResourceNames"
+          DependsOnTargets="$(CustomizeXlfSourceNamesDependsOn)"
           BeforeTargets="PrepareResourceNames;GatherXlf"
+          Condition="'$(SkipCustomizeXlfSourceNames)'!='true'"
```

Comment:
You shouldn't need both this Condition and the change to the DependsOnTargets - if a Condition evaluates to `false` then the DependsOnTargets should not be run at all. Were you seeing different?